flexmeasures/ui/templates/crud/asset_properties.html

```diff
@@ -0,0 +1,379 @@
+                                <div class="form-group">
+                                    <label for="assset-type" class="col-sm-6 control-label">Asset Type</label>
+                                    <div class="col-md-6">
+                                        <input class="form-control" id="asset-type-id" name="asset-type" type="text"
+                                            value="{{ asset.generic_asset_type.name }}" disabled></input>
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <label for="asset-id" class="col-sm-6 control-label">Asset id</label>
+                                    <div class="col-md-6">
+                                        <input class="form-control" id="asset-id" name="asset-id" type="text"
+                                            value="{{ asset.id }}" disabled></input>
+                                    </div>
+                                </div>
+
+                                <div class="form-group">
+                                    {{ asset_form.attributes.label(class="col-sm-3 control-label") }}
+                                    <div class="col-md-3">
+                                        {{ asset_form.attributes(class_="form-control") }}
+                                        {% for error in asset_form.errors.attributes %}
+                                        <span style="color: red;">[{{error}}]</span>
+                                        {% endfor %}
+                                    </div>
+                                </div>
+
+                                <div class="form-group">
+                                    <label class="control-label">Location</label>
+                                    <small>(Click map to edit latitude and longitude in form)</small>
+                                    <div id="mapid"></div>
+                                    <button class="btn btn-sm btn-responsive btn-success create-button" type="submit"
+                                        value="Save"
+                                        style="margin-top: 20px; float: right; border: 1px solid var(--light-gray);">
+                                        Save
+                                    </button>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </form>
+                </div>
+            </div>
+            {% endif %}
+   
+        </div>
+        <div class="col-md-8">
+            <div class="sensors-asset card">
+                <h3>Asset Properties - {{ asset.name }}</h3>
+                <div class="row">
+                    <div class="col-md-7">
+                        <div class="table-responsive">
+                            <table class="table table-striped nav-on-click w-100 mx-auto" title="View Asset Properties">
```

Comment:
This table should not have the fancy smart table capabilities like "Filter records", sorting, pagination

Make it a simple table, and also vertical, much like the one on the right for attributes. With two columns, "Property" and "Value".

(Maybe `render_attributes()` can be used for that or copied?)